spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StdCmdConfig.scala

```diff
@@ -28,25 +28,27 @@ import scala.util.matching.Regex
  * Even if a field is mandatory it needs a default value.
  */
 case class StdCmdConfig(
-                      cmdLineArgs: Array[String],
-                      datasetName: String = "",
-                      datasetVersion: Int = 1,
-                      reportDate: String = "",
-                      reportVersion: Option[Int] = None,
-                      rawFormat: String = "xml",
-                      menasCredentialsFactory: MenasCredentialsFactory = InvalidMenasCredentialsFactory,
-                      charset: Option[String] = None,
-                      rowTag: Option[String] = None,
-                      csvDelimiter: Option[String] = None,
-                      csvHeader: Option[Boolean] = Some(false),
-                      csvQuote: Option[String] = None,
-                      csvEscape: Option[String] = None,
-                      cobolOptions: Option[CobolOptions] = None,
-                      fixedWidthTrimValues: Option[Boolean] = Some(false),
-                      performanceMetricsFile: Option[String] = None,
-                      rawPathOverride: Option[String] = None,
-                      folderPrefix: Option[String] = None,
-                      persistStorageLevel: Option[StorageLevel] = None)
+                         cmdLineArgs: Array[String],
+                         datasetName: String = "",
+                         datasetVersion: Int = 1,
+                         reportDate: String = "",
+                         reportVersion: Option[Int] = None,
+                         rawFormat: String = "xml",
+                         menasCredentialsFactory: MenasCredentialsFactory = InvalidMenasCredentialsFactory,
+                         charset: Option[String] = None,
+                         rowTag: Option[String] = None,
+                         csvDelimiter: Option[String] = None,
+                         csvHeader: Option[Boolean] = Some(false),
+                         csvQuote: Option[String] = None,
+                         csvEscape: Option[String] = None,
+                         cobolOptions: Option[CobolOptions] = None,
+                         fixedWidthTrimValues: Option[Boolean] = Some(false),
+                         performanceMetricsFile: Option[String] = None,
+                         rawPathOverride: Option[String] = None,
+                         folderPrefix: Option[String] = None,
+                         persistStorageLevel: Option[StorageLevel] = None,
+                         failOnInputNotPerSchema: Boolean = false
```

Comment:
Should we name the variable the same as the command line option, e.g. `strictSchemaCheck`?

Reply:
I was thinking about it. Won't it make the reader expect that any divination will cause exception? Missing and extra columns won't.